lib/block-patterns.php

```diff
@@ -62,3 +62,32 @@
 						<!-- /wp:columns -->',
 	)
 );
+
+$core_block_patterns = array(
+	'media-text-nature',
+	'two-images-gallery',
+	'cover-quote',
+	'cover-columns',
+	'three-columns-media-text',
+	'quote-portrait',
+	'hero-section',
+	'media-text-art',
+	'text-two-columns-title',
+	'three-columns-list',
+	'text-two-columns-title-2',
+	'heading',
+	'three-images-gallery',
+	'two-columns-list',
+	'media-text-arquitecture'
+);
+
+foreach ( $core_block_patterns as $core_block_pattern ) {
+	register_block_pattern(
+		'core/' . $core_block_pattern,
+		require __DIR__ . '/block-patterns/' . $core_block_pattern . '.php'
+	);
+}
+
+register_block_pattern_category( 'nature', array( 'label' => _x( 'Nature', 'Block pattern category' ) ) );
```

Comment:
I think the categories should be a bit more functional: text, columns, galleries, hero / cover, etc. Can we revise them?

Reply:
Agreed — I've left some category recommendations [below](https://github.com/WordPress/gutenberg/pull/29973#issuecomment-801940348).